src/app/logic/player.js

```diff
@@ -1,23 +1,37 @@
+// eslint-disable-next-line import/extensions
+import { DIFFICULTY_LEVELS } from '../data/consts.js';
+// eslint-disable-next-line import/extensions
+import { calculatePoints } from './calculatePoints.js';
+
 export class Player {
   constructor() {
     this.correctAnswersScore = 0;
     this.detailQuestionData = [];
   }
 
-  checkAndRegisterAnswer(playerAnswer, correctAnswer) {
+  checkAndRegisterAnswer(
+    playerAnswer,
+    correctAnswer,
+    difficultyLevelsProperty,
+  ) {
     if (!playerAnswer) {
       throw new Error(
         'error with given argument: playerAnswer, the value is incorrect, it should be string',
       );
     }
-    let isAnswerCorrect = true;
-    if (playerAnswer === correctAnswer) {
-      this.correctAnswersScore += 1;
-      isAnswerCorrect = true;
-    } else {
-      isAnswerCorrect = false;
+    const isAnswerCorrect = playerAnswer === correctAnswer;
+
+    if (
+      DIFFICULTY_LEVELS[difficultyLevelsProperty].calculatingPoints !==
```

Comment:
There is an extra check here about the diffcultyLevels, perhaps you cuold do it in calculatiingPoints function? Its as if part of the logic was here, part there.

Reply:
here it is checked if level is easy or different, because, if it is easy, then Player just checks if answer is correct and thats all. If something else, then is invoked calculatePoints() and registerAnswer().
Maybe make one check in Player and divide calculatePoints() to two independent functions (for different levels)?